devtools/platform-descriptor-json/src/main/resources/bundled-codestarts/tooling/dockerfiles/codestart.yml

```diff
@@ -2,3 +2,17 @@
 name: dockerfiles
 type: tooling
 preselected: true
+language:
+  base:
+    data:
+      dockerfile:
+        jvm:
+          from: registry.access.redhat.com/ubi8/ubi-minimal:8.1
+          java-package: java-11-openjdk-headless
```

Comment:
shouldn't this be tied to what version of java user targeted ?

Reply:
I have no idea, this is taken from the legacy codegen.

Reply:
okey, separate issue then.